DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@model SearchableDelegateViewModel
+@{
+  var activeTagCss = Model.IsActive ? "nhsuk-tag" : "nhsuk-tag nhsuk-tag--grey";
+  var selfRegTagCss = "nhsuk-tag nhsuk-tag--grey";
+  var adminTagCss = "nhsuk-tag nhsuk-tag--green";
+  var passwordTagCss = Model.IsPasswordSet ? "nhsuk-tag nhsuk-tag--green" : "nhsuk-tag nhsuk-tag--red";
+}
+
+@*TODO: Search and sort functionality is part of HEEDLS-491.
+  Filename includes 'Searchable' to avoid having to change name later*@
+
+<div class="nhsuk-expander-group" id="@Model.Id-card">
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.Id-name" name="name" role="heading" aria-level="2">

```

Comment:
This shouldn't have the role="heading" aria-level="2". Was raised as a bug for 415 and fixed for admin cards in 416.